src/main/java/seedu/taassist/logic/commands/AssignCommand.java

```diff
@@ -63,8 +64,13 @@ public CommandResult execute(Model model) throws CommandException {
 
         studentsToAssign.forEach(s -> model.setStudent(s, s.addModuleClass(moduleClassToAssign)));
 
-        String indexOrIndices = indices.size() == 1 ? "index" : "indices";
-        return new CommandResult(String.format(MESSAGE_SUCCESS, indexOrIndices, indices, moduleClassToAssign));
+        return new CommandResult(getSuccessMessage(studentsToAssign, moduleClassToAssign));
+    }
+
+    public static String getSuccessMessage(List<Student> students, ModuleClass moduleClass) {
+        String studentNames = students.stream().map(student -> student.getName().toString())
+                .collect(Collectors.joining(", "));
+        return String.format(MESSAGE_SUCCESS, moduleClass, studentNames);
```

Comment:
I think it is possible to abstract this method to `StringUtil` too. However, this is a trivial matter and can be done in v1.4 so don't worry.

Reply:
Nice idea, I managed to do it although it looks a bit hacky